.github/NIGHTLY_BUILD_FAILURE.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "nightly build failed"
+assignees: kobyhallx, phated, tomafrench
```

Comment:
Can you add me here too -- I'd like to be notified when nightly builds fail too!